cmd/otelcol/config/collector/logs_config_linux.yaml

```diff
@@ -0,0 +1,740 @@
+  # For more information about memory limiter, see
+  # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiter/README.md
+  memory_limiter:
+    check_interval: 2s
+    limit_mib: ${SPLUNK_MEMORY_LIMIT_MIB}
+
+
+exporters:
+  # Enables the Splunk HEC exporter.
+  # Full configuration here: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/splunkhecexporter
+  splunk_hec:
+    # Splunk HTTP Event Collector token.
+    token: "${SPLUNK_HEC_TOKEN}"
+    # URL to a Splunk instance to send data to.
+    endpoint: "${SPLUNK_HEC_URL}"
+    # Optional Splunk source: https://docs.splunk.com/Splexicon:Source
+    source: "otel"
+    # Optional Splunk source type: https://docs.splunk.com/Splexicon:Sourcetype
+    sourcetype: "otel"
+    # Profiling data should always be sent to Splunk Observability Cloud which is configured in the next section.
+    profiling_data_enabled: false
+
+  # Enables the Splunk HEC exporter to send profiling data to Splunk Observability Cloud.
+  splunk_hec/profiling:
+    # Splunk Observability Cloud access token.
+    token: "${SPLUNK_ACCESS_TOKEN}"
+    # Splunk Observability Cloud endpoint.
+    endpoint: "${SPLUNK_INGEST_URL}/v1/log"
+    # Disabling logs as only profiling data should be sent to Splunk Observability Cloud.
+    log_data_enabled: false
+
+
+
+extensions:
+  health_check:
+    endpoint: "${SPLUNK_LISTEN_INTERFACE}:13133"
+
+  memory_ballast:
+    # In general, the ballast should be set to 1/3 of the collector's memory.
+    # The simplest way to specify the ballast size is set the value of SPLUNK_BALLAST_SIZE_MIB env variable.
+    size_mib: ${SPLUNK_BALLAST_SIZE_MIB}
+
+  # Storage extension for storing filelog checkpoints.
+  # Checkpoints allow the receiver to pick up where it left off in the case of a
+  # collector restart.
+  file_storage/filelogs:
+    # Location where checkpoint files are stored. This directory must exist and
+    # must be writable by the collector..
+    directory: /var/lib/otelcol/filelogs
+    compaction:
+      on_start: true
+      directory: /tmp/
+
+
+service:
+  extensions: [health_check, memory_ballast, file_storage/filelogs]
+  pipelines:
+    logs:
+      receivers:
+        #- filelog/apache-access
+        #- filelog/apache-error
+        #- filelog/cassandra
+        #- filelog/cassandra-debug
+        #- filelog/cassandra-output
+        #- filelog/docker
+        #- filelog/etcd
+        #- filelog/jetty9
+        #- filelog/jetty9-debug
+        #- filelog/jetty9-request
+        #- filelog/mongodb
+        #- filelog/mysql-error
+        #- filelog/mysql-query
+        #- filelog/mysql-slow_query
+        #- filelog/nginx-access
+        #- filelog/nginx-error
+        #- filelog/postgresql
+        #- filelog/rabbitmq
+        #- filelog/rabbitmq-startup
+        #- filelog/rabbitmq-startup_err
+        #- filelog/redis
```

Comment:
Unfortunately no, we get periodic messages (info level): "finding files: no files match the configured criteria"

Reply:
Oh, that's sad. It makes the OOTB experience worse. Do you think that's the right behavior? Maybe worth bringing to the collector community as an issue for discussion.

Reply:
I think that is an acceptable *default* behavior -- if you misconfigure the receiver, you get a warning instead of wondering why logs are not coming in. We can propose a new configuration setting for the filelog receiver that controls this.